packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -459,6 +462,18 @@ class DraggableDetails {
   final Offset offset;
 }
 
+/// Represents the details when a pointer event occurred on the [DragTarget].
+class DragTargetDetails {
```

Comment:
The usability of the new API would probably be slightly more ergonomic if this object would also include the data. Then you only have to listen to one callback.

Reply:
I was concerned that including the data in the new callback would confusion about using one callback vs another. But I am happy to move forward with this suggestion.

If the updated new callback contains both data and details, should the name change to imply that it does the same, plus more, as [onAccept]? I.e. should I change the new callback name from [onAcceptDetails] to , e.g. [onAcceptWithDetails]?

Reply:
From a usability point of view it would be nicer if you only have to deal with one callback. We should make it very clear in the documentation for both callbacks that one is the strict super-set of the other. We could even disallow setting both at the same time to make that absolutely clear.  

Naming this callback is ... hard. `onAcceptWithDetails` is my favorite.